From 5811de2de52f91a7e77bfbbb95fe670d07a1eb27 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Mon, 27 May 2024 18:43:33 +0530
Subject: [PATCH 61/98] va: Fix libdrm include

The libdrm/ prefix should not be used, it will be provided by the
pkgconfig file. Also HAVE_LIBDRM is necessary.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/6932>
---
 gst-libs/gst/va/gstvavideoformat.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gst-libs/gst/va/gstvavideoformat.h b/gst-libs/gst/va/gstvavideoformat.h
index 43b61352bf67..327478a374ca 100644
--- a/gst-libs/gst/va/gstvavideoformat.h
+++ b/gst-libs/gst/va/gstvavideoformat.h
@@ -25,8 +25,8 @@
 #include <gst/video/video.h>
 #include <va/va.h>
 
-#ifndef G_OS_WIN32
-#include <libdrm/drm_fourcc.h>
+#ifdef HAVE_LIBDRM
+#include <drm_fourcc.h>
 #else
 /**
  * DRM_FORMAT_INVALID: (skip) (attributes doc.skip=true)
-- 
2.43.0

